app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -933,6 +1195,11 @@ class RecentlyPlayedFragmentTest {
     }
   }
 
+  fun setUpTestFragment(activity: RecentlyPlayedActivity) {
+    activity.supportFragmentManager.beginTransaction().add(testFragment, TEST_FRAGMENT_TAG)
+      .commitNow()
```

Comment:
```suggestion
    activity.supportFragmentManager
      .beginTransaction()
      .add(testFragment, TEST_FRAGMENT_TAG)
      .commitNow()
```

Wrapping should start at the first ``.`` starting the chain of operations.